# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clip/linux-2.6.15.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur clip-prg-1.1.15-1-orig/clip/_date.c clip-prg-1.1.15-1/clip/_date.c
--- clip-prg-1.1.15-1-orig/clip/_date.c	2005-12-07 08:46:59.000000000 +0000
+++ clip-prg-1.1.15-1/clip/_date.c	2006-02-06 00:32:19.835452750 +0000
@@ -554,7 +554,7 @@
 	if (n & 2)
 		d += tm.tms_stime;
 
-	d /= CLK_TCK;
+	d /= CLOCKS_PER_SEC;
 	_clip_retndp(mp, d, 10, 2);
 	return 0;
 }
diff -Nur clip-prg-1.1.15-1-orig/clip/clip.c clip-prg-1.1.15-1/clip/clip.c
--- clip-prg-1.1.15-1-orig/clip/clip.c	2005-08-25 08:36:33.000000000 +0000
+++ clip-prg-1.1.15-1/clip/clip.c	2006-02-06 00:29:03.987213000 +0000
@@ -1003,8 +1003,8 @@
 	clock_t diff = end - beg;
 	int /*m, */ s, ms;
 
-	s = diff / CLK_TCK;
-	ms = (diff % CLK_TCK) * 100 / CLK_TCK;
+	s = diff / CLOCKS_PER_SEC;
+	ms = (diff % CLOCKS_PER_SEC) * 100 / CLOCKS_PER_SEC;
 	/*m=s/60;
 	   s=s%60; */
 	sprintf(buf, "%d.%02ds", s, ms);
